packages/frontapp/src/app/components/CategorySearch/index.tsx

```diff
@@ -144,8 +222,32 @@ const CategorySearch = ({ setActive }: any) => {
                 />
               </OneCategory>
             ))}
+            {showMore &&
+              item.subAttributes
+                .slice(7, item.subAttributes.length)
+                .map((attribute) => (
+                  <OneCategory key={attribute.id}>
+                    {attribute.title}{' '}
+                    {attribute.numberOfItems && `- ${attribute.numberOfItems}`}
+                    <StyledCheckBox
+                      type={item.inputType}
+                      name={item.title.toLowerCase()}
+                      value={attribute.title}
+                      onChange={(e) =>
+                        radioBtnHandler(
+                          item.subAttributes,
+                          item.inputType,
+                          e.target.value
+                        )
+                      }
+                      onMouseDown={() => checkBoxHandler(attribute)}
+                    />
+                  </OneCategory>
+                ))}
             {item.subAttributes.length > 7 && (
```

Comment:
I know that is not you changed but can you please replace this magic number '7'?